posts/2022-12-06-22.0.0.13-beta.adoc

```diff
@@ -0,0 +1,353 @@
+
+
+Similar to Jakarta EE 10 features, we've also been releasing various MicroProfile 6 features in the recent beta releases, and the 22.0.0.13-beta marks the first release that includes all of MicroProfile 6!  Also provided in this beta release is the ability to automatically purges FFDC log files after they reach a configured age.
+
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
+// important because it is supposed to grab the readers attention.  Make sure to keep the blank lines 
+//
+// Throughout the doc, replace 22.0.0.13-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+The link:{url-about}[Open Liberty] 22.0.0.13-beta includes the following beta features (along with link:{url-prefix}/docs/latest/reference/feature/feature-overview.html[all GA features]):
+
+* <<jakarta10, Jakarta EE 10 Platform and Web Profile>> 
+** <<authentication, Application Authentication 3.0 (Jakarta Authentication 3.0)>>
+** <<authorization, Application Authorization 2.1 (Jakarta Authorization 2.1)>>
+** <<security, Application Security 5.0 (Jakarta Security 3.0)>>
+** <<faces, Jakarta Faces 4.0>>
+* <<mp6, MicroProfile 6.0>>
+* <<maxFfdcAge, Configurable maximum FFDC age>>
+
+
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23358
+// Contact/Reviewer: ReeceNana,jhanders34
+// // // // // // // // 
+[#jakarta10]
+== Jakarta EE 10 Platform and Web Profile
+
+In the Open Liberty 22.0.0.9-beta, we made available Jakarta EE 10 Core Profile and ever since we've been hard at work releasing more Jakarta EE 10 features with each subsequent beta.  We're proud to announce that the 22.0.0.13-beta is the culmination of that effort and makes available Jakarta EE 10.0 Application Client, Jakarta EE Web Profile 10.0, and Jakarta EE Platform 10.0.
+
+Liberty provides convenience features for running all of the component specifications that are contained in the Jakarta EE Web Profile and Jakarta EE Platform.  `webProfile-10.0` and `jakartaee-10.0` Liberty features provide the Jakarta EE Web Profile and Jakarta EE Platform version 10 Liberty features respectively.  For Jakarta EE 10 features in the application client, use the `jakartaeeClient-10.0` Liberty feature.
+
+
+Using these convenience features enables developers to rapidly develop applications using all of the APIs contained in the Web Profile and Platform specifications.  Unlike the `webProfile-9.1` Liberty feature, the `webProfile-10` Liberty feature does not enable the XML Binding feature, because XML Binding is not a Web Profile component specification.  If your application relies on XML Binding APIs, you must explicitly enable the `xmlBinding-4.0` Liberty feature when using the `webProfile-10.0` Liberty feature.

```

Comment:
```suggestion
Using these convenience features enables developers to rapidly develop applications using all of the APIs contained in the Web Profile and Platform specifications.  Unlike the `webProfile-9.1` Liberty feature, the `webProfile-10` Liberty feature does not enable the XML Binding feature, because XML Binding is not a Web Profile component specification. If your application relies on XML Binding APIs, you must explicitly enable the `xmlBinding-4.0` Liberty feature when using the `webProfile-10.0` Liberty feature.

```